### PR TITLE
Added improvement for CaseStudy edit view for related pages

### PR DESCRIPTION
**Pull request by webbyfox**
Improvement of edit screen of CaseStudy edit view

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/1341
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -60,6 +59,7 @@
     'django.contrib.humanize',
     'sso',
     'core.apps.CoreConfig',
+    'wagtail.admin',
```
Comment (stevejalim) [inline config/settings.py:62]: Why did it need to move? if it's important (eg loading order) it may be worth a comment

Comment (webbyfox) [inline config/settings.py:62]: Moved to fix template lookup in our app

File: config/urls.py [hunk 2]
```diff
@@ -56,4 +56,4 @@
     # Serve static and media files from development server
     urlpatterns += staticfiles_urlpatterns()
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
-    urlpatterns += [path('__debug__/', include(debug_toolbar.urls))]
+    urlpatterns = [path('__debug__/', include(debug_toolbar.urls))] + urlpatterns
```
Comment (webbyfox) [inline config/urls.py:59]: fix debug toolbar as all urls were 404

Comment (stevejalim) [inline config/urls.py:59]: Curious why this was needed (not that it's a problem!)

File: core/static/wagtailadmin/js/page-chooser.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    ModalWorkflow({
+      url: initialUrl,
+      urlParams: urlParams,
+      onload: PAGE_CHOOSER_MODAL_ONLOAD_HANDLERS,
+      responses: {
+        pageChosen: function (pageData) {
+          input.val(pageData.id)
+          openAtParentId = pageData.parentId
+          pageTitle.text(pageData.title)
+          // Added next two lines to amend class to element for related pages
```
Comment (stevejalim) [inline core/static/wagtailadmin/js/page-chooser.js:36]: 💅 👍

File: core/templates/wagtailadmin/widgets/page_chooser.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+{% load static %}
+
+{% block chosen_state_view %}
+    <span class="title">{{ page.get_admin_display_title }}</span>
+{% endblock %}
+
+{% block edit_chosen_item_url %}{% if page %}{% url 'wagtailadmin_pages:edit' page.id %}{% endif %}{% endblock %}
+
+
+<script type="text/javascript" src="{% static '/static/wagtailadmin/js/page-chooser.js'%}"></script>
```
Comment (stevejalim) [inline core/templates/wagtailadmin/widgets/page_chooser.html:11]: Worth a comment about why this file has been customised?

**Review by stevejalim — APPROVED** [review]
LGTM - some questions but nothing blocking

File: config/settings.py (hunk 1 above)
Comment (stevejalim) [reply config/settings.py:62]: Worth a comment to stop someone 'tidying up' and breaking things?

**Pull request merged**